vipul/CSS_Assignment_6.txt

```diff
@@ -30,4 +29,8 @@ Scenario B
         </body>
 </html>
 
+<<<<<<< HEAD
+-> Scenario A before Amber Academy text 20px left margin will be display after that text with background color yellow. In scenario B text Amber Academy with padding left 20px with background color yellow will be display.
+=======
 -> Scenario A before Amber Academy text 20px left margin will be display after that text with background color yellow.In scenario B text Amber Academy with padding left 20px with background color yellow will be display.
+>>>>>>> master

```

Comment:
move all assignment file to the seperate folder

Reply:
i want to delete this file. how to do that? this file is already pushed